src/numberGenerator.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const numberGenerator = () => {
+  const number = [];
```

Comment:
Using a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) instead of an array would reduce complexity of this generation, because `number.includes` looks through all the items in array until it finds one, Set's `.has` checks for needed item immediately.

```suggestion
  const number = new Set();
```